packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -74,3 +74,58 @@
     }
   }
 }
+
+@mixin iui-file-uploaded-template {
+  width: 100%;
+  height: 100%;
+  line-height: normal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: var(--iui-color-background-backdrop);
```

Comment:
Why did you add background color?
![image](https://user-images.githubusercontent.com/83585998/214849007-d678ffa6-0972-43df-b305-30450d6ee6f1.png)